tasks/config_virtual_networks.yml

```diff
@@ -1,39 +1,38 @@
 ---
-- name: config_virtual_networks | defining virtual networks
+
+- name: config virtual_networks | define virtual networks
+  become: true
   virt_net:
     command: define
     name: "{{ item['name'] }}"
     xml: "{{ lookup('template', 'vm-network.xml.j2') }}"
-  register: kvm_bridges_defined
-  become: true
-  with_items: "{{ kvm_virtual_networks }}"
+  loop: "{{ kvm_virtual_networks }}"
 
-- name: config_virtual networks | setting state of virtual networks
+- name: config virtual networks | setting state of virtual networks
+  become: true
   virt_net:
-    state: "{{ item['state'] }}"
     name: "{{ item['name'] }}"
-  become: true
-  with_items: "{{ kvm_virtual_networks }}"
+    state: "{{ item['state'] }}"
   when: item['state'] is defined
+  loop: "{{ kvm_virtual_networks }}"
 
-- name: config_virtual networks | setting virtual networks to autostart
+- name: config virtual networks | setting virtual networks to autostart
+  become: true
   virt_net:
-    autostart: yes
+    autostart: "{{ item['autostart'] }}"
```

Comment:
Let's set this one to `true` as we do in the next task for `false`.

Reply:
Essentially, that's what happens. The conditions state:
```
  virt_net:
    autostart: "{{ item['autostart'] }}"
    name: "{{ item['name'] }}"
  when:
     - item['autostart'] is defined
     - item['autostart']
     - (item['state'] | default("")) == "active"
```

So based on the conditions, the only value that can be in `item['autostart']` is `true`.

What strikes me as odd is that you probably use this role to reconfigure existing kvm networks (set it from `active` -> `inactive`)... and a special omission kicks in that case, if you also set `autostart: true`.

Would you, please, describe your typical usage of this role in the full extent within the README.md file? Maybe it is possible to infer a simpler API.

Thanks.

Btw. I pushed the change.

Reply:
Agreed, just for consistency because of the next task using the same `item['autostart']` but it was pre-defined as `false`.

Reply:
OK, I modified the code. Feel free to merge at your convenience.